docs/sink-configuration.md

```diff
@@ -110,6 +110,17 @@ The following options are available:
 * `tags` - FIXME. Default. `none`
 * `storeEvents` - Control storage of events. Default: `true`
 
+### OpenTSDB
+This sink supports monitoring metrics and events.
+To use the opentsdb sink add the following flag:
+
+    --sink=opentsdb:<OPENTSDB_SERVER_URL>
+
+Currently, accessing opentsdb via its rest apis doesn't need any authentication, so you
```

Comment:
The opentsdb sink uses bluebreezecf/opentsdb-goclient to store k8s metrics and events into opentsdb v2.
Currently opentsdb v2 lacks an authentication and access control system. Therefore no authentication is required when accessing the API.(http://opentsdb.net/docs/build/html/api_http/index.html#authentication-permissions).